### PR TITLE
Update metadata, funding, rors, etc.

### PR DESCRIPTION
**Pull request by rowanc1**

### CODE REVIEW

File: myst.yml [hunk 1]
```diff
@@ -3,8 +3,11 @@ version: 1
 project:
   id: msa-em-article-template-1
   title: Article template for Elemental Microscopy
+  description: An article template for the Elemental Microscopy journal focused on educational and informative content.
```
Comment (rowanc1) [inline myst.yml:6]: This is shown on the main page.

File: myst.yml [hunk 2]
```diff
@@ -3,8 +3,11 @@ version: 1
...
   short_title: EM Template
-  keywords: []
+  keywords:
+    - Article Template
+    - Computational Article
```
Comment (rowanc1) [inline myst.yml:11]: Feel free to tweak

File: myst.yml [hunk 3]
```diff
@@ -17,7 +20,13 @@ project:
   #      url: 'http://localhost:8888'
   #      token: '512ac78f14e1141db1fac17e8b4099c1e5bc7d589518b38c'
   open_access: true
+  license: CC-BY-4.0
   banner: banner.png
+  thumbnail: thumbnail.webp
+  date: 2024-07-24
+  history:
+    submitted: 2023-06-09
+    published: 2024-07-24
```
Comment (rowanc1) [inline myst.yml:30]: We can change this when we update the DOI.

File: myst.yml [hunk 4]
```diff
@@ -36,14 +46,28 @@ project:
...
+    - statement: We thank the Microscopy Society of America for sponsoring this project.
+      recipients:
+        - cophus
+        - garnavides
+    - statement: Work at the Molecular Foundry was supported by the Office of Science, Office of Basic Energy Sciences, of the U.S. Department of Energy under Contract No. DE-AC02-05CH11231.
+      id: DE-AC02-05CH11231
+      sources:
+        - name: Office of Science, Office of Basic Energy Sciences, of the U.S. Department of Energy
+      recipients:
+        - cophus
```
Comment (rowanc1) [inline myst.yml:71]: This is currently repeated in the text, but necessary here.

File: myst.yml [hunk 5]
```diff
@@ -60,5 +84,3 @@ project:
     STEM: scanning transmission electron microscopy
 site:
   template: book-theme
-  domains:
-    - microscopy-EM000000.curve.space
```
Comment (rowanc1) [inline myst.yml:64]: No longer necessary.

**Review by gvarnavi — APPROVED** [review]

**Pull request merged**